src/plugins/plot/MctPlot.vue

```diff
@@ -425,6 +396,7 @@ export default {
     });
   },
   beforeUnmount() {
+    this.resetAlignment();
```

Comment:
Let's move this to an `onBeforeUnmount()` hook in the composable itself. This would eliminate the need to call it explicitly in every component that uses it. Instead, it would be called automatically on component unmount for any component using it.

Reply:
The reason I didn't put it in the composable is because the reset only needs to happen at the top level (parent). The children wouldn't reset the alignment state.